docs/api-designer/getting-started-api-designer/README.adoc

```diff
@@ -115,16 +114,16 @@ Use the {product-long-api-designer} web console to create an OpenAPI or AsyncAPI
 
 ifndef::qs[]
 .Prerequisites
-* You're logged in to the {product-api-designer} web console at {service-url-api-designer}[^].
+* You're logged in to the {product-long-api-designer} web console at {service-url-api-designer}[^].
 endif::[]
 
 .Procedure
-. In the {service-url-api-designer}[{product-api-designer} web console], click *Create a schema or API design*.
+. In the {service-url-api-designer}[{product-long-api-designer} web console^], click *Create a schema or API design*.
 . Complete the form to provide the details of the new design:
```

Comment:
Per IBM style (and our recent MAS style discussions), this lead-in sentence for what is essentially a list should probably be a complete sentence?

For example, I recently updated the _Create a topic_ procedure in the service binding guide to do exactly that: https://github.com/redhat-developer/app-services-guides/blob/main/docs/kafka/service-binding-kafka/README.adoc?plain=1#L477

Reply:
I thought it was a complete sentence! I've rephrased it as follows -- is this rewrite better: 
"Complete the form to provide the following details for the new design:"

Reply:
Closing this comment as resolved -- please reopen if you don't agree :)